webviz_config/webviz_store.py

```diff
@@ -175,6 +179,8 @@ def build_store(self):
                     output.to_parquet(f"{path}.parquet")
                 elif isinstance(output, pathlib.Path):
                     shutil.copy(output, f"{path}{output.suffix}")
+                elif isinstance(output, io.BytesIO):
+                    open(f"{path}", "wb").write(output.getvalue())
```

Comment:
Remove `f`-string since the string `path` is used directly?

Perhaps use `pathlib.Path.write_bytes()` in order to make sure the file is properly closed.